photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/search/GaussianProcessSearch.scala

```diff
@@ -92,17 +92,33 @@ class GaussianProcessSearch[T](
         // hyperparameter spaces.
         val kernel = new Matern52
 
+        //Finding the overall bestEval
```

Comment:
Missing space after `//`.

Also, we should include a brief explanation that prior evals are expected to be mean-centered, and that's why we're doing this.

Reply:
Done